slack-base/src/main/java/com/hubspot/slack/client/models/ReactionIF.java

```diff
@@ -0,0 +1,22 @@
+package com.hubspot.slack.client.models;
+
+import java.util.List;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ReactionIF {
+  @JsonProperty("name")
+  String getName();
+  @JsonProperty("count")
+  Integer getCount();
+  @JsonProperty("users")
+  List<String> getUser();
```

Comment:
Should we rename this to `getUsers`? I imagine the list is returned when multiple users reacted with the same emoji.

Reply:
Also, we won't need `JsonProperty` annotation in this case.